config/config.properties

```diff
@@ -469,13 +469,13 @@ extendeds3.configUri=http://localhost:9020?identity=user&secretKey=password
 
 ##region DurableLog Settings
 
-# A Metadata Checkpoint is a special internal write inside the DurableLog (persisted to Tier1 DurableDataLog) that takes
+# A Metadata Checkpoint is a special internal write inside the DurableLog (persisted to Durable Data Log) that takes
```

Comment:
I'm aware that is better to replace Tier 2 -> Long Term Storage for clarity. But I'm not sure whether making Tier 1 to disappear as well is making things clearer or not (is an open question).

Reply:
We've agreed to eliminate the Tier 1 term as well. DurableLog/DurableDataLog is the term to use to be consistent with the code.

Reply:
So, I'm leaving it as is. @RaulGracia / @andreipaduroiu hope that is OK.